### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Andrew77788**
- [DEMO LINK](https://Andrew77788.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Andrew77788.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,36 @@
...
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+    <title> </title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
```
Comment (volodymyr-soltys97) [inline src/index.html:17]: Need to use the semantic tag for header
```suggestion
    <header class="header">
```

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,36 @@
...
+      <div class="logo"><img src="images/logo.png" alt="logo"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:18]: The same
```suggestion
      <a class="logo">
        <img src="images/logo.png" alt="logo">
      </a>
```

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+      <ul class="navbar_menu">
+        <li class="for_line">
+          <a href="#" class="menu_1">Apple</a>
+          <div class="line"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:22]: Redundant, you need to add this link as pseudoelement 
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -1,14 +1,36 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <div class="logo"><img src="images/logo.png" alt="logo"></div>
+      <ul class="navbar_menu">
+        <li class="for_line">
+          <a href="#" class="menu_1">Apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:21]: Need to fix classes everywhere, also add the nav tag here
```suggestion
    <nav class="nav">
      <ul class="nav__list">
        <li class="nav__item">
          <a href="#" class="nav__link">Apple</a>
```

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,36 @@
...
+          <div class="line"></div>
+        </li>
+        <li><a href="#" class="menu_2">Samsung</a></li>
```
Comment (volodymyr-soltys97) [inline src/index.html:24]: Need to fix code style and classes everywhere
```suggestion
        <li class="nav__item">
          <a href="#" class="nav__link">Samsung</a>
        </li>
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,78 @@
...
+  text-align: center;
+  color: black;
+  margin: 10px;
+}
+
+/* у зразку було підчеркування виділеного елементу, але не впевнений, що це треба було робити, можливо тільки на Figma
+/* a:hover {
+  border-bottom: solid 1px blue;
+  color:var(--color_selection) ;
+} */
```
Comment (volodymyr-soltys97) [inline src/style.css:46]: Need to remove comments
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,78 @@
...
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 40px;
+}
+
+a {
```
Comment (volodymyr-soltys97) [inline src/style.css:35]: Use a classes selector

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,78 @@
...
+}
+
+.menu_1 {
+  color: var(--color-selection);
+
+}
+
+.line {
+  position: absolute;
+  left: 50%;
```
Comment (volodymyr-soltys97) [inline src/style.css:71]:
```suggestion
  left: 0;
```

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,78 @@
...
+  transform: translate(-50%, 0);
```
Comment (volodymyr-soltys97) [inline src/style.css:72]:
```suggestion
```

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,78 @@
...
+  bottom: 0 ;
+  border-radius: 2px;
+  height: var(--w-font);
+  width: 37px;
```
Comment (volodymyr-soltys97) [inline src/style.css:76]:
```suggestion
  width: 100%;
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
Need to remove this static color for this link
image

Also need to add the blue color for all links when hovering
image

Need to remove the left padding for the first link
image

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,72 @@
...
+
+li {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  position: relative;
+}
+
+li:not(:last-child) {
```
Comment (nazarmatsevych) [inline src/style.css:67]: Don't use tags for styling, use classes instead

**Review by nazarmatsevych — APPROVED** [review]
Good work!

**Pull request closed without merging**